client/src/components/SignIn/style.css

```diff
@@ -131,7 +131,7 @@ input {
 }
 
 /* helper text styling on hover */
-.input-field .helper-text:hover {
+#signup-input .helper-text:hover {
```

Comment:
changing the id's like impacts these css rules

Reply:
I know it does. that's why I did it . lol. one background is white and the other is dark so without the duplicate ids the text color on one form or the other is overwritten since .input-field is a materialize class selector that can only be overridden via id if there are more than two style variations (including the default materialize.css styling).